### PR TITLE
lesson6

### PR DESCRIPTION
**Pull request by xRuTeRx**

### CODE REVIEW

File: lesson6/task2/server/server.go [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+)
+
+func handler(w http.ResponseWriter, r *http.Request) {
+	// send Not found in such case
+	if r.URL.Path != "/" {
+		http.NotFound(w, r)
+		return
+	}
+	switch r.Method {
+	case "GET":
```
Comment (denislituev) [inline lesson6/task2/server/server.go:23]: better to use constants from http package here
see https://pkg.go.dev/net/http#pkg-constants

File: lesson6/task2/server/server.go [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+		fmt.Fprint(w, message)
+	case "POST":
```
Comment (denislituev) [inline lesson6/task2/server/server.go:25]: better to use constants from http package here
see https://pkg.go.dev/net/http#pkg-constants

File: lesson6/task3/server/server.go [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+)
+
+func handler(w http.ResponseWriter, r *http.Request) {
+	// send Not found in such case
+	if r.URL.Path != "/" {
+		http.NotFound(w, r)
+		return
+	}
+	switch r.Method {
+	case "GET":
```
Comment (denislituev) [inline lesson6/task3/server/server.go:17]: please replace with constants

File: lesson6/task3/server/server.go [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+		body, _ := ioutil.ReadFile("../form/form.html")
+		fmt.Fprint(w, string(body))
+
+	case "POST":
```
Comment (denislituev) [inline lesson6/task3/server/server.go:21]: same here

**Review by denislituev — CHANGES_REQUESTED** [review]

File: lesson6/task2/server/server.go [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+}
+
+func main() {
+	// server port number
+	const port = 8080
+
+	fmt.Printf("Launching server on port: %d \n\n", port)
+
+	// set handler for route '/'
+	http.HandleFunc("/", handler)
```
Comment (nikonok) [inline lesson6/task2/server/server.go:47]:
```
TCP Server (S) listen for a connections on a port
TCP Client (C) connects to a TCP server on a port 
```
TCP, not HTTP protocols, see my examples one more time=)

**Review by nikonok — CHANGES_REQUESTED** [review]

**Review by denislituev — APPROVED** [review]

**Review by nikonok — APPROVED** [review]

**Pull request merged**